Branch-SDK/src/main/java/io/branch/referral/BillingGooglePlay.kt

```diff
@@ -257,6 +257,6 @@ class BillingGooglePlay private constructor() {
             .addContentItems(contentItems)
             .logEvent(context)
 
-        PrefHelper.Debug("Successfully logged in-app purchase as Branch Event")
+        BranchLogger.v("Successfully logged in-app purchase as Branch Event")
```

Comment:
This might be good to have under `.i` for clients testing integrations since this is a feature. Assuming we don't want them to have to raise their logging level to `.v` (problem solving)
cc @nsingh-branch

Reply:
I missed this, I actually did replace all with patterns. But I will again go through each message and fix its type.